Common/VR/PPSSPPVR.cpp

```diff
@@ -560,12 +560,11 @@ void PreprocessSkyplane(GLRStep* step) {
 
 	// Clear sky with the fog color.
 	if (!vrCompat[VR_COMPAT_FBO_CLEAR]) {
-		GLRRenderData skyClear {};
-		skyClear.cmd = GLRRenderCommand::CLEAR;
+		GLRRenderData &skyClear = step->commands.insert(step->commands.begin());
+		skyClear.cmd = GLRRenderCommand::CLEAR;  // intentional zero-initialize
```

Comment:
Hm, this value (`CLEAR`) isn't zero so this comment seems like it might be out of place.  Not sure if the scissor values need setting?

-['Unknown]

Reply:
Oh yeah, that's a left-behind comment. gone. Scissor should probably be added, but this is Lubos' VR hack, I'm not touching it right now.

Reply:
Actually never mind, we have scissor in the clear command, so we probably really should initialize it... ugh.

Reply:
#17546